### DISCUSSION TITLE
Custom authenticator not created

### DISCUSSION THREAD

**Discussion by carlosProsperoware** [serverless/serverless#10983] [Q&A] [answered]
Hi I am trying to create a custom authenticator using the next code:

provider:
  name: aws
  runtime: nodejs14.x
  baseParams: ${file(params-base.yml)}
  region: ${opt:region, 'us-east-1'}
  httpApi:
    authorizers:
      customAuthorizer:
        type: request
        functionName: CFLambdaAzureAuthorizer
        identitySource:
          - $request.header.Authorization	

but when package the cloud formation generated is :

![image](https://user-images.githubusercontent.com/95829747/164118188-5252704c-ac3c-4c72-855a-0f6be5a64d8a.png)

It creates a jwt instead of a request.

I am using framework version 1.68.0.
Thanks

**Comment by pgrzesik** [comment, marked as answer]
Hello @carlosProsperoware - you're using a heavily outdated version of the Framework - it's definitely not supporting custom authorizers - did you get any warnings in the output of the command? Is there anything preventing your from upgrading Framework version?

**Reply by carlosProsperoware** [reply]
Thansk a lot for your answer, 
I did not got any warnigs, but I have upgraded and of course as you said it worked!
Thanks again for your help!

**Reply by pgrzesik** [reply]
Happy to hear you've been able to resolve your issues @carlosProsperoware 🙌
